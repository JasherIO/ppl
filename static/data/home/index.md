---
templateKey: home-post
tabs:
  - name: Players
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus facilisis, nisl quis porta bibendum, nulla purus vestibulum sem, sit amet ultrices nisi elit ut odio. Sed rutrum eu velit at rhoncus.'
  - name: Teams
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus facilisis, nisl quis porta bibendum, nulla purus vestibulum sem, sit amet ultrices nisi elit ut odio. Sed rutrum eu velit at rhoncus.'
  - name: Format
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus facilisis, nisl quis porta bibendum, nulla purus vestibulum sem, sit amet ultrices nisi elit ut odio. Sed rutrum eu velit at rhoncus.'
sponsors:
  - img: /img/AllYours.png
    link: 'https://google.com'
    name: All Yours Buffet
  - img: /img/WhatASave.png
    link: 'https://google.com'
    name: What a Save Esports
  - img: /img/EggingGrocers.png
    link: 'https://google.com'
    name: Egging Grocers
---

