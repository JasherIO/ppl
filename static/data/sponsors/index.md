---
templateKey: sponsor-post
sponsors:
  - img: /img/AllYours.png
    link: 'https://google.com'
    name: All Yours Buffet
  - img: /img/WhatASave.png
    link: 'https://google.com'
    name: What a Save Esports
  - img: /img/EggingGrocers.png
    link: 'https://google.com'
    name: Egging Grocers
---

