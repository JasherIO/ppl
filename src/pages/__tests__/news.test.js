import React from "react"
import renderer from "react-test-renderer"

import data from './posts.json'
import NewsPage from '../news/index'

jest.mock('../../components/News-Cards', () => () => (<></>))

describe("NewsPage", () =>
  it("renders correctly", () => {
    const tree = renderer
      .create(<NewsPage data={data} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  }))