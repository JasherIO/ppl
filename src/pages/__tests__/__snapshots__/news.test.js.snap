// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsPage renders correctly 1`] = `
<div
  className="container"
>
  <section
    className="section"
  >
    <div
      className="level is-mobile"
    >
      <div
        className="level-left"
      >
        <div
          className="level-item"
        >
          <h1
            className="title is-3"
          >
            Recent News
          </h1>
        </div>
      </div>
    </div>
  </section>
</div>
`;
